docs/source/simulators/UnitarySimulator.rst

```diff
@@ -0,0 +1,19 @@
+Unitary Simulator
+=================
+
+The :code:`UnitarySimulator` takes a given quantum circuits and combines the individual gates/operations into one unitary.
+
+
+Usage in Python
+###############
+
+Available backend in the provider :code:`DDSIMProvider`:
+
+- :code:`unitary_simulator` to simulate and sample from the resulting state
```

Comment:
the result of the unitary simulator is not a state 🙃 and it is not sampled from.

Reply:
Ups, copy-paste mistake